### DISCUSSION TITLE
Cask "Wolfram Engine" v14.1 is available to download but its livecheck has not been updated

### DISCUSSION THREAD

**Discussion by singularitti** [Homebrew/discussions#5568] [Casks] [answered]
### Output of `brew config`

```shell
HOMEBREW_VERSION: 4.3.17-91-g4482e25
ORIGIN: https://github.com/Homebrew/brew
HEAD: 4482e25cb27efb4482da1273d2998c2935d69424
Last commit: 12 hours ago
Core tap JSON: 23 Aug 05:31 UTC
Core cask tap JSON: 23 Aug 05:31 UTC
HOMEBREW_PREFIX: /opt/homebrew
HOMEBREW_CASK_OPTS: []
HOMEBREW_EDITOR: code
HOMEBREW_MAKE_JOBS: 10
HOMEBREW_SORBET_RUNTIME: set
Homebrew Ruby: 3.3.4 => /opt/homebrew/Library/Homebrew/vendor/portable-ruby/3.3.4_1/bin/ruby
CPU: 10-core 64-bit arm_firestorm_icestorm
Clang: 15.0.0 build 1500
Git: 2.46.0 => /opt/homebrew/bin/git
Curl: 8.7.1 => /usr/bin/curl
macOS: 14.6.1-arm64
CLT: 15.3.0.0.1.1708646388
Xcode: N/A
Rosetta 2: false
```

### Output of `brew doctor`

```shell
Your system is ready to brew.
```

### Description of issue

Wolfram Engine v14.1 is available on its website, but the [livecheck](https://github.com/Homebrew/homebrew-cask/blob/37d03cf3b9ba8c391b77520d863cf8466c5b9f0a/Casks/w/wolfram-engine.rb#L12) in its corresponding cask still shows [14.0.0.0](https://files.wolframcdn.com/packages/Homebrew/latest.txt). Who should we contact to fix this?

<img width="1197" alt="image" src="https://github.com/user-attachments/assets/d93f2057-de2a-499c-9d19-0db786f39a46">

### Relevant casks

wolfram-engine

**Comment by SMillerDev** [comment]
> Who should we contact to fix this?

https://docs.brew.sh/Updating-Software-in-Homebrew

**Comment by daeho-ro** [comment]
Here is the PR
- https://github.com/Homebrew/homebrew-cask/pull/183253

**Comment by singularitti** [comment]
Thanks!

**Reply by daeho-ro** [reply]
Unfortunately, the PR is closed without merge.
The application is not signed and this might be the reason to separate the homepage version and homebrew.
Sorry for that, you may contact to Wolfram.

**Reply by daeho-ro** [reply, marked as answer]
Recover the PR, changed the livecheck url only, and there was a new version for homebrew users and so it works now.

**Reply by singularitti** [reply]
Great, thank you everyone!

**Comment by miccal** [comment]
Have a look at my version of `wolfram-engine` from my tap for a working `livecheck`: https://github.com/miccal/homebrew-miccal/blob/master/Casks/m-wolfram-engine.rb:
```
|-> brew livecheck --debug m-wolfram-engine
/opt/homebrew/Library/Homebrew/brew.rb (Formulary::NullLoader): loading m-wolfram-engine
/opt/homebrew/Library/Homebrew/brew.rb (Cask::CaskLoader::FromNameLoader): loading m-wolfram-engine

Cask:             m-wolfram-engine
Livecheckable?:   Yes

URL:              https://account.wolfram.com/dl/WolframEngine?&platform=Mac
Strategy:         HeaderMatch
Regex:            /WolframEngine\/(\d+(?:\.\d+)+)\//i

Matched Versions:
14.1.0.0

m-wolfram-engine: 14.1.0.0 ==> 14.1.0.0
```
